### PR TITLE
Add showing path on cropped image

### PR DESCRIPTION
**Pull request by wangyung**
implement #2

### CODE REVIEW

File: clayview/src/main/java/com/linecorp/clay/view/ClayView.kt [hunk 1]
```diff
@@ -192,15 +200,37 @@ class ClayView : BaseTransformableView {
...
             bitmap?.let { srcBitmap ->
+                val padding = (selectionDrawable.strokeWidth / resources.displayMetrics.density / 2).toInt()
                 val selectedPathOnBitmap = Path(currentDrawingPath.path)
                 selectedPathOnBitmap.transform(getInvertMatrix(theDisplayMatrix))
-                return cropImage(source = srcBitmap, path = selectedPathOnBitmap,
-                                 antiAlias = antiAlias)
+                val croppedImage = cropImage(source = srcBitmap, path = selectedPathOnBitmap,
+                        antiAlias = antiAlias, padding = padding)
+                if (drawPathOnCroppedImage) {
+                    val tranlateMatrix = Matrix()
```
Comment (leesunghyun) [inline clayview/src/main/java/com/linecorp/clay/view/ClayView.kt]: typo : tranlateMatrix -> translateMatrix

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -4,4 +4,5 @@
   <string name="use_xml">Use xml</string>
   <string name="undo">Undo</string>
   <string name="trim">Trim</string>
+  <string name="showPath">Show path after trimming</string>
```
Comment (leesunghyun) [inline app/src/main/res/values/strings.xml]: showPath -> show_path?

File: clayview/src/main/java/com/linecorp/clay/drawable/PathSelectDrawable.kt [hunk 3]
```diff
@@ -19,10 +19,12 @@ import com.linecorp.clay.view.DrawingPath
  * Path selection drawable, if highlight, it draws clear inside the path and draw mask outside
  * If not highlight, it just draw the path
  */
-internal class PathSelectDrawable(var drawingPath: DrawingPath, @ColorInt maskColor: Int) : Drawable() {
+internal class PathSelectDrawable(var drawingPath: DrawingPath,
```
Comment (leesunghyun) [inline clayview/src/main/java/com/linecorp/clay/drawable/PathSelectDrawable.kt]: It looks like this class header formatting is a little different from the guidelines.
https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting

**Review by leesunghyun — APPROVED** [review]
LGTM!

File: clayview/src/main/java/com/linecorp/clay/drawable/PathSelectDrawable.kt (hunk 3 above)
Comment (wangyung) [reply clayview/src/main/java/com/linecorp/clay/drawable/PathSelectDrawable.kt]: yes, i will update it later

**Comment by wangyung** [comment]
merge it after fixing some typo

**Pull request merged**